index.html

```diff
@@ -45,6 +45,238 @@ <h4>i am a software developer</h4>
         </li>
       </ul>
     </section>
+    <section class="work-section">
+      <h1>My Recent Works</h1>
+      <div class="img-container">
+        <img src="./images/image1.png" alt="" />
+        <h2>Multi-Posts Stories</h2>
+        <p>
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard dummy text ever
+          since the 1500s, when an unknown printer took a standard dummy text.
+        </p>
+        <ul>
+          <li>
+            <a href="#">css</a>
+          </li>
+          <li>
+            <a href="#">html</a>
+          </li>
+          <li>
+            <a href="#">Bootstraps</a>
+          </li>
+          <li>
+            <a href="#">Ruby</a>
+          </li>
+        </ul>
+        <button>see project</button>
+      </div>
+    </section>
+    <section class="cards">
```

Comment:
- [OPTIONAL] As requested by the previous code reviewer, it will be nice to place your cards within the works section as this will make your code more organized and semantic. Please kindly place your  cards as child elements of the works section.